src/pages/AnalyticsPage/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,41 @@
+import config from 'config'
+
+import { useFlags } from 'shared/featureFlags'
+import Badge from 'ui/Badge'
+import TabNavigation from 'ui/TabNavigation'
+
+function Tabs() {
+  const { codecovAiFeaturesTab } = useFlags({
+    codecovAiFeaturesTab: false,
+  })
+
+  return (
+    <TabNavigation
```

Comment:
lots of repeated components here. I wonder if we can pull it out into a common component

Reply:
Is this exactly the same as the Header component?

Reply:
Yep

Reply:
Im on board to make it a common component

Reply:
https://github.com/codecov/engineering-team/issues/2546